docs/src/content/ignition/docs/guides/upgradeable-proxies.md

```diff
@@ -227,6 +227,9 @@ module.exports = demoModule;
 Next it's time to upgrade our proxy to a new version. To do this, we'll create a new file within our `ignition/modules` directory called `UpgradeModule.js` (or `UpgradeModule.ts` if you're using TypeScript). Inside this file, we'll again break up our module into two parts. To start, we'll write our `UpgradeModule`:
 
 ```js
+const { buildModule } = require("@nomicfoundation/hardhat-ignition/modules");
+const proxyModule = require("./ProxyModule");
```

Comment:
@wesleysmyth it was assumed that a user would be able to infer this much on their own, however being explicit is not necessarily a bad thing. If we add these lines, then we also need a typescript version that uses import instead. You can look elsewhere in this file for a reference on what the formatting should look like.